### DISCUSSION TITLE
Error: expected Undefined to be a GraphQl schema while accessing Home page after logging in.

### DISCUSSION THREAD

**Discussion by AnnuCode** [prisma/prisma#14148] [Q&A] [answered]
Hi, I am facing an error after I try to access the Home page after logging in. I have used JWT authentication and the error pops up only while the cookie exists, it disappears when I delete the cookie, hence I can't login. Please note that all the queries/resolvers are working fine in Apollo playground. 
Built using Next.js, GraphQl, Prisma, Apollo.

GitHub repo is [here](https://github.com/AnnuCode/hackernews).
Details to log in:
**Username**: neymar  **Password**: trickster
To reproduce the error you can click on _Hackernews_  after logging in to try to go to Home.

**Error: Expected undefined to be a GraphQL schema.**

According to me, possible mistakes committed by me are: 
-  An unhandled error somewhere
- resolver or type in feed.ts graphql Type might be the culprit(though it works fine in the Apollo playground)

_It will be very helpful if someone can solve this issue_

**Comment by pantharshit00** [comment]
Hello @AnnuCode , our apologies for the delay in getting back to you! Due to the large number of questions in our community channels, unfortunately this slipped through the cracks. It's very important for us to address your question, so let us know in a comment if this is still an open discussion. 

I am unable to reproduce this:
<img width="1506" alt="image" src="https://user-images.githubusercontent.com/22195362/194480065-bf86a59e-b716-4076-b738-8b8983fe1f42.png">

Can you please let me know if this is still relevant and occurs?

**Reply by vladi-stevanovic** [reply]
👋 Hello @AnnuCode I'm going through pending GitHub Discussions to check if you need any help. Is this issue still open?
Please let us know if we can investigate it further or if you've already solved it, we'd love to know how!

It would be super useful for other users facing this problem! 🙌

**Comment by AnnuCode** [comment, marked as answer]
Hello @vladi-stevanovic and @pantharshit00 apologies for the late reply. The above error "Error: Expected undefined to be a GraphQL schema" was caused by Nexus while trying to make a custom scalar type (date-time). The error vanished when I used a 3rd party library to create the date-time scalar('graphql-iso-date').

**Reply by vladi-stevanovic** [reply]
Thank you for getting back to us and sharing the cause of the issue! 🙏

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
